### ISSUE TITLE
useWebSocket is ignored

### ISSUE THREAD

**Issue by miyconst** [Palindrom/palindrom-client#3]
```
<puppet-client use-web-socket="false" usewebsocket="false" ref="root"></puppet-client>
```

Still uses WebSocket for communication.

**Comment by miyconst** [comment]
@warpech could you please fix it?

There is a bug in Chrome, which causes an error when using WS.
http://stackoverflow.com/questions/37066865/one-or-more-reserved-bits-are-on-reserved1-0-reserved2-1-reserved3-1

**Comment by warpech** [comment]
This should be auto-fixed with #9

**Comment by miyconst** [comment]
No, it's not fixed. @joozek78 please fix it.

**Comment by warpech** [comment]
Yeah, I meant it should be automatically fixed if #9 is accepted and merged, which it was not yet. https://github.com/puppetjs/puppetjs/issues/85 has bigger priority than this now

**Comment by warpech** [comment]
@joozek78 now that we decided to abandon https://github.com/PuppetJs/puppet-polymer-client/issues/9, this issue needs to be fixed properly. `pupper-polymer-client` needs to wrap `puppet. useWebSocket` property.

When this is fixed, the "Use WebSocket" checkbox in `starcounter-debug-aid` will start to function again.

**Comment by joozek78** [comment]
This is actually caused by how Polymer properties work. Take a look at [this thread](https://github.com/Polymer/polymer/issues/1812#issuecomment-112226630). There's even a work-around proposed there

**Comment by warpech** [comment]
Check if this is still relevant in the Polymer 2.0 version.

**Comment by warpech** [comment]
The correct attribute name is `use-web-socket`. There is indeed problem with converting attribute values to booleans. The fix is in PR https://github.com/Palindrom/palindrom-client/pull/84

**Comment by warpech** [comment]
Fixed in #84
